### ISSUE TITLE
[rush] pnpm list not working in rush repository

### ISSUE THREAD

**Issue by NicolasDuran** [microsoft/rushstack#2689]
## Summary

 Running pnpm list in any project of my rush monorepo doesn't output anything

```
➜  a git:(master) pnpm list
➜  a git:(master) pnpm list -r
➜  a git:(master) 
```

## Repro steps

I created this git repository: https://github.com/NicolasDuran/rush-debug as a demo of the bug

1. Clone
2. `rush install`
3. `cd apps/a && pnpm list`

  **Expected result:** 

```
➜  a git:(master) npm ls
a@ /***/rush-debug/apps/a
└── express@4.17.1 -> /***/rush-debug/common/temp/node_modules/.pnpm/express@4.17.1/node_modules/express
```

  **Actual result:** 

```
➜  a git:(master) pnpm list   
➜  a git:(master) 
```

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@microsoft/rush` globally installed version? | 5.46.1 |
| `rushVersion` from rush.json? | 5.46.1 |
| `useWorkspaces` from rush.json? | true |
| Operating system? | Linux |
| Would you consider contributing a PR? | Yes |
| Node.js version (`node -v`)? | v15.14.0 |

**Comment by iclanton** [comment]
Interesting. I would expect it to work in a repo with workspaces enabled. @D4N14L, any thoughts?

**Comment by D4N14L** [comment]
Likely doesn't work due to not being run from the `common/temp` dir. We have a proposal that I've been meaning to find time for over here that would solve this: https://github.com/microsoft/rushstack/issues/2347

**Comment by octogonz** [comment]
Closing since this is a duplicate of #2347

**Comment by octogonz** [comment]
🚀The new `rush-pnpm` command is released with  `@microsoft/rush` 5.74.0.  

To use it:  Besides upgrading `rushVersion` in your **rush.json**, you need to add the `rush-pnpm` shell command by doing `npm install --global @microsoft/rush`.
